library/std/src/Std/Math.qs

```diff
@@ -9,6 +9,7 @@ import Std.Diagnostics.*;
 // Constants PI, E, LogOf2.
 //
 
+/// # Summary
```

Comment:
This may have been an artifact of how the files were created/moved from the originals, in most files the first callable still had summary and it was the rest that got lost.